src/TextUI/TestRunner.php

```diff
@@ -490,6 +490,9 @@ class_exists($arguments['printer'], false)) {
                 if ($arguments['coverageText'] == 'php://stdout') {
                     $outputStream = $this->printer;
                     $colors       = $arguments['colors'];
+                    if($colors == PHPUnit_TextUI_ResultPrinter::COLOR_NEVER) {
```

Comment:
why not just `$colors       = $arguments['colors'] && !PHPUnit_TextUI_ResultPrinter::COLOR_NEVER`?